core/rawdb/taiko_l1_origin.go

```diff
@@ -52,16 +69,40 @@ func WriteL1Origin(db ethdb.KeyValueWriter, blockID *big.Int, l1Origin *L1Origin
 	}
 }
 
-// ReadL1Origin retrieves the given L2 block's L1Origin from database.
+// DeleteL1Origin removes the L1Origin.
+func DeleteL1Origin(db ethdb.KeyValueWriter, blockID *big.Int) {
```

Comment:
What happened when delete the `head l1origin`?

Reply:
We inserted the newest `l1origin id` by `WriteHeadL1Origin`, but it will reach the null pointer after deleting the data.